docs/en/reference/information_schema/character_sets.md

```diff
@@ -0,0 +1,16 @@
+---
+displayed_sidebar: "English"
+---
+
+# character_sets
+
+`character_sets` identifies the character sets available.
+
+The following fields are provided in `character_sets`:
+
+| **Field**            | **Description**                                              |
+| -------------------- | ------------------------------------------------------------ |
+| CHARACTER_SET_NAME   | The character set name.                                      |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/588fd7dd-d083-44a2-aee7-4cfb3429f7be)
-> The name of the character set.